op-e2e/system/fjord/check_scripts_test.go

```diff
@@ -27,11 +27,11 @@ func TestCheckFjordScript(t *testing.T) {
 		fjordActivation *hexutil.Uint64
 		expectErr       bool
 	}{
-		{
-			name:            "fjord_activated",
-			fjordActivation: &genesisActivation,
-			expectErr:       false,
-		},
+		// {
+		// 	name:            "fjord_activated",
+		// 	fjordActivation: &genesisActivation,
+		// 	expectErr:       false,
+		// },
```

Comment:
@sebastianst @ajsutton would love your input here

```
Error:      	Received unexpected error:
            	            	fastLz: gas-price-oracle: expected GasPriceOracle address 0xa919894851548179A0750865e7974DA599C0Fac7 does not match actual address 0xc0d3C0d3C0d3c0D3C0D3C0d3C0d3C0D3C0D3000f
            	Test:       	TestCheckFjordScript/fjord_activated
            	Messages:   	should not error on CheckAll
```

1. This test case errors out in develop branch as well
2. It is not showing up in CI because it's parallel execution (if disable parallel execution, you'll see the error)
3. I tried debugging into the code, everything is at least few years old, and don't see any recent changes that could cause this.

So two questions here:
1. It's weird that parallel execution cause somehow neglect the test error
2. Any suggestions on how to deal with this? This code seems to only work under op-chain-ops which is not part of the critical path

Reply:
This specifically is due to the genesis state having fjord already active. There is some ongoing work now to make the allocs setup more flexible cc @mslipper. It could be easy to just skip this for now with `t.Skip`. It is horrifying that parallel execution causes this to be missed

Reply:
Thanks, after merging latest change from develop, this error goes away in CI (although it still exists). 

Not changing anything related to this for now

Reply:
Awesome, merged your fix in